languages/elm/maintainers.md

```diff
@@ -4,8 +4,7 @@ These awesome people help maintain the Elm track.
 
 ## Senior Maintainers
 
-TODO: add senior maintainers
```

Comment:
Why did you do this?

Reply:
This was in the rust maintainers.md, which is where I initially copied the template from. I'm happy to take it out if you want.

Reply:
Oh I see, I deleted it. It was in the rust one, but maybe not in the c# one. I have literally no opinion on this. Just tell me what you want.

Reply:
Leave `TODO: add senior maintainers` in

Reply:
Hi Adam, I've pushed the change for this.
Cheers, Cedd